src/azure-cli/azure/cli/command_modules/storage/operations/account.py

```diff
@@ -443,3 +443,36 @@ def update_blob_service_properties(cmd, instance, enable_change_feed=None, enabl
         instance.restore_policy = cmd.get_models('RestorePolicyProperties')(
             enabled=enable_restore_policy, days=restore_days)
     return instance
+
+
+def create_encryption_scope(cmd, client, resource_group_name, account_name, encryption_scope_name,
+                            encryption_key_source=None, encryption_key_uri=None):
+    EncryptionScope = cmd.get_models('EncryptionScope')
+
+    if encryption_key_source:
+        encryption_scope = EncryptionScope(source=encryption_key_source)
+
+    if encryption_key_uri:
+        EncryptionScopeKeyVaultProperties = cmd.get_models('EncryptionScopeKeyVaultProperties')
+        encryption_scope.key_vault_properties = EncryptionScopeKeyVaultProperties(key_uri=encryption_key_uri)
+
+    return client.put(resource_group_name=resource_group_name, account_name=account_name,
+                      encryption_scope_name=encryption_scope_name, encryption_scope=encryption_scope)
+
+
+def update_encryption_scope(cmd, client, resource_group_name, account_name, encryption_scope_name,
+                            encryption_key_source=None, encryption_key_uri=None, state=None):
+    EncryptionScope, EncryptionScopeState = cmd.get_models('EncryptionScope', 'EncryptionScopeState')
+
+    if encryption_key_source:
+        encryption_scope = EncryptionScope(source=encryption_key_source)
+
+    if encryption_key_uri:
+        EncryptionScopeKeyVaultProperties = cmd.get_models('EncryptionScopeKeyVaultProperties')
+        encryption_scope.key_vault_properties = EncryptionScopeKeyVaultProperties(key_uri=encryption_key_uri)
+
+    if state is not None:
+        encryption_scope.state = EncryptionScopeState(state)
```

Comment:
Same as above

Reply:
key source will always has default value and they have validator to check usage.